### DISCUSSION TITLE
How can I force update my meilisearch database?

### DISCUSSION THREAD

**Discussion by EricTheMagician** [danny-avila/LibreChat#1802] [Help Wanted]
I noticed my melisearch container was down for quite a while and I have a lot of my search queries fail because the database was not populated when I created new chats. Is there a way to have meilisearch reindexed?

**Comment by danny-avila** [comment]
Yup! Just restart the main server process (“LibreChat”). Indices are checked on startup

**Reply by EricTheMagician** [reply]
Thanks for the super prompt response.
I can confirm that this worked!
